packages/ui/src/memberships/components/SignOutButton.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { SignOutIcon } from '@/common/components/page/Sidebar/LinksIcons/SignOutIcon'
+
+import { TextSmall } from '../../common/components/typography'
+import { BorderRad, Colors, Transitions } from '../../common/constants'
+
+interface Props {
+  onClick: () => void
+}
+
+export const SignOutButton = ({ onClick }: Props) => {
```

Comment:
Instead of adding a `SignOutButton` with the same style as `AddMembershipButtonSwitch` could you add something like `@/common/components/buttons/DetailsButtons` and a story for it, which would be used like this:
```tsx
<DetailsButtons title="New Member" icon={<MyProfileIcon />} disabled={isTransactionPending} onClick={() => { ... }}>
  Create a New Membership
</DetailsButtons>
{!!active && (
  <DetailsButtons title="Sign Out" icon={<SignOutIcon />} onClick={() => { ... }}>
    Sign out of the active Membership
  </DetailsButtons>
)}
```